app/src/main/java/com/example/bottomnav/Parse.java

```diff
@@ -9,18 +9,19 @@
 import org.eclipse.cdt.internal.core.dom.parser.cpp.*;
 
 import java.io.*;
-
 import java.io.ByteArrayOutputStream;
 import java.io.InputStream;
 import java.util.ArrayList;
 import java.util.HashMap;
-import java.util.Locale;
 import java.util.Map;
 import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
 public class Parse {
     public HashMap<String, ConstContents> constRepo = new HashMap<>();
     public HashMap<String, ArrayList<StructContents>> structRepo = new HashMap<>();
-    public HashMap<String, PayloadMap> idStruct = new HashMap<>();
+    public HashMap<String, ArrayList<String>> idStruct = new HashMap<>();
```

Comment:
Since there are two `String`s in the type parameters, I recommend having a comment that defines the meaning of each string - for example `// CAN ID -> List<CAN Struct>`